bodhi/client/__init__.py

```diff
@@ -985,6 +985,90 @@ def info_release(name, url, **kwargs):
         print_release(res)
 
 
+@releases.group(name='critpath')
+def critpath_pkgs():
+    # Docs that show in the --help
+    """Interact with critical path packages of a release."""
+    # Developer Docs
+    """Manage the critical path packages of a release."""
+    pass  # pragma: no cover
+
+
+@critpath_pkgs.command(name='list')
+@handle_errors
+@click.argument('name')
+@url_option
+@staging_option
+def list_critpath_packages(name, url, **kwargs):
+    """
+    List critical path packages of a release.
+
+    NAME: Release name (e.g. F27)
+    """
```

Comment:
For this one we do want to show both developer docs (argument types) and user docs (what you have here), similar to what we've done on other functions in this file.